db/models.py

```diff
@@ -1,5 +1,11 @@
+from datetime import datetime
+
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
+import settings
```

Comment:
it's better to use
`from django.conf import settings` because there may be a few settings or a folder named "settings"